collaborative_filtering_tutorial_in_r.Rmd

```diff
@@ -0,0 +1,848 @@
+
+```{r setup, include=FALSE}
```

Comment:
- The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

- If the file contains a setup chunk in .Rmd file, it should not contain a setup label. (The bookdown render will fail if there are duplicate chunk labels.)
- i.e. use {r, include=FALSE} instead of {r setup, include=FALSE}.
- [See sample .Rmd](https://github.com/jtr13/cc21/blob/main/sample_project.Rmd)